presidio-anonymizer/presidio_anonymizer/anonymizer_parameters/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""Initializing all the existing anonymizers parameters."""
```

Comment:
Is this branch created over the V2_anonymizers_improvements?

Reply:
No, since I won't change the engine. Cherry picked what I needed for now. Notice that it's a draft